introduction.md

```diff
@@ -7,3 +7,9 @@ I kung fu hard and code even harder.
 Sometimes I use my sharp blade to slash my code sometimes I use my skills to avenge for justice.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?